libwacom/libwacom-database.c

```diff
@@ -592,6 +655,29 @@ libwacom_parse_tablet_keyfile(const char *datadir, const char *filename)
 		g_strfreev (string_list);
 	}
 
+	string_list = g_key_file_get_string_list (keyfile, DEVICE_GROUP, "EDIDMatch", NULL, NULL);
+	if (string_list) {
+		GArray *array;
+		guint i;
+
+		array = g_array_new (FALSE, FALSE, sizeof(WacomEDID));
+		device->num_edid = 0;
+
+		for (i = 0; string_list[i]; i++) {
+			WacomEDID match;
+
+			if (!edid_match_from_string(string_list[i], &match)) {
+				DBG("failed to match '%s' for EDID. Skipping.\n", string_list[i]);
```

Comment:
I'd suggest using "failed to parse" instead to make it clear that this is an issue with the datafile, not the actual matching process.